boards/starfive/visionfive2_jh7110/visionfive2_jh7110_defconfig

```diff
@@ -0,0 +1,16 @@
+# SPDX-License-Identifier: Apache-2.0
+
+
+# Zephyr Kernel Configuration
+CONFIG_PRINTK=y
+CONFIG_XIP=n
+
+# Serial Drivers
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_UART_NS16550=y
```

Comment:
Continuing the discussion from https://github.com/zephyrproject-rtos/zephyr/pull/69359#discussion_r1502495926.

The SiFive UART is not enabled for HiFive Unmatched. You're likely referencing the `CONFIG_UART_SIFIVE_PORT_n` options - note that these are additional symbols. The Kconfig symbol for the driver itself is `CONFIG_UART_SIFIVE`, and is enabled by the `DT_HAS_SIFIVE_UART0_ENABLED` dts-generated config.

Reply:
Updated the file accordingly